frontend/src/components/ViewActivities/activityfns.js

```diff
@@ -44,22 +38,28 @@ export function compareActivities(a, b) {
   return -1;
 }
 
-
 /**
- * Get the field of field name `fieldName` from `activity  or the default value.
- * 
- * @param {Object} activity Activity to get field from.
- * @param {string} fieldName Name of the field in the activity to get. 
- * @param defaultValue Default value to return if activity[fieldName] can't be found. 
- * Can be any type.
- * @param {string} prefix The prefix to put before a returned value if the field exists.
- * @returns `activity[fieldName]` if possible, else `defaultValue`. Can be any type.
+ * Sort a list of trip activities by date. 
+ * @param {Array} tripActivities Array of activities.
+ * @return List of trip activities in the form
```

Comment:
Add type {}?

Reply:
That'll come in my jsdoc PR because it requires a typedef.